tensilelite/Tensile/Common.py

```diff
@@ -1028,6 +1028,9 @@ def getArchitectureName(gfxName):
 
     # in mix mode F8 need to convert to F16, do this before(0) ds or after(1) ds
     "ConvertAfterDS":              [False, True],
+
+    # Force disable shadow init to release more sgpr in preloop
+    "ForceDisableShadowInit":      [False, True],
```

Comment:
Suggest not to add new kernel argument for just shawdoinit setup

Reply:
how much performance difference if we force this on?

Reply:
Can't generate some kernels without this flag.